yarn-project/noir-contracts/scripts/transform_json_abi.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+# Sadly, yarn-project expects ABI files in a format different to what is output from nargo.
+# This provides a tool to quickly do the transform with jq.
+# It's not currently used as the transform is done in TypeScript when we generate the contract wrapper.
+# TODO: Why don't our contract classes just work with noir abis?
```

Comment:
The noir compiler itself does not know about aztec abis, since aztec-nargo exists anyway it should probably know this 

issue required

Reply:
https://github.com/AztecProtocol/aztec-packages/issues/3812 guess this is close enough?